js/scripts.js

```diff
@@ -0,0 +1,132 @@
+/* v2 */
+$(function () {
+	var globResp;
+
+	var url = 'https://restcountries.eu/rest/v1/name/';
+	var $countriesList = $('#countries');
+
+	$('#search').on('click', searchCountries);
+	$('body').on('click', 'li', expandField); // hehe....
+
+	function searchCountries() {
+
+		$countriesList.empty();
+		var countryName = $('#country-name').val();
+
+		// sprawdzenie na długosć
+		if (countryName.length < 2) {
+			alert("Prosze wpisac co najmniej 2 znaki w pole wyszukiwania");
+			return false // przerywamy funkcje...
+		} else {
+			//
+			$.ajax({
+				url: url + countryName,
+				method: 'GET',
+				success: function (resp) {
+					// robie tak dziko bo chcem przekazać  countryName w
+					//parametrze
```

Comment:
zrób tą funkcję oddzielnie i tutaj ją tylko odpal. Skoro potrzebujesz `countryName` to je przekaż w wywołaniu funkcji.